.github/workflows/main.yml

```diff
@@ -25,16 +25,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ros_distro: [rolling, iron, humble, foxy]
+        ros_distro: [rolling, iron, humble, foxy, jazzy]
         include:
         - ros_distro: 'rolling'
-          os: ubuntu-22.04
+          os: ubuntu-24.04
```

Comment:
Why iron is changed to U24?

Reply:
I think you meant for 'rolling'. Rolling distro is actually targeted for U24. Till U24 became stable, we were using U22. https://docs.ros.org/en/rolling/Releases/Release-Rolling-Ridley.html#currently-supported-platforms

Reply:
Got it Tnx